CONTRIBUTING.md

```diff
@@ -1 +1,51 @@
-admin-secure-payment-content: "Utilizamos o Stripe, serviço de pagamento já consolidado no mercado, para efetuar as transações com segurança"
+#7
```

Comment:
You would like to reference the issue here, but it's not the place, please fix it.